openfga/v1/authzmodel.proto

```diff
@@ -52,19 +52,11 @@ message TypeDefinition {
 }
 
 message Relation {
-  string name = 1 [(validate.rules).string = {
```

Comment:
I don't think we should get rid of 'name', because if you pass around a Relation object and want to get the name of the relation, then you can't. Let's leave this.

Reply:
I don't like `name` here because yesterday I had a rough time figuring out why something didn't work, and it was because I had a typo between the map key and what was in the Relation. This can only occur, I suppose, if you are typing stuff in using the JSON, but it still makes me a bit uneasy.

In any case, for the sake of going forward, I'll add it back in.